starter_code/src/movies.js

```diff
@@ -2,18 +2,96 @@
 
 // Iteration 1: All rates average - Get the average of all rates with 2 decimals 
 
- 
+function ratesAverage(moviesArr) {
+    const sumRates = moviesArr.reduce(function (accumulator, value) {
+        return accumulator + parseFloat(value.rate);
+    }, 0) / moviesArr.length;
```

Comment:
I would divide by moviesArr.length after the reduce function.

so when you return